recipes/rapidcheck/all/conanfile.py

```diff
@@ -33,32 +37,23 @@ class RapidcheckConan(ConanFile):
         "enable_gtest": False,
     }
 
```

Comment:
Would be great to have `package_type` here https://docs.conan.io/2/reference/conanfile/attributes.html?highlight=package_type#package-type